README.md

```diff
@@ -9,7 +9,7 @@ Experimental support for [qb-core](https://github.com/qbcore-framework/qb-core)
 
 # Config
 
-Refer to the [documentation](https://overextended.github.io/docs/ox_inventory/) setting your config.
+Refer to the [documentation](https://overextended.github.io/docs/ox_inventory) setting your config.
```

Comment:
This should also redirect to the new docs

Reply:
my bad

Reply:
This is the right link to be straight to the ox_inventory doc :D
https://overextended.dev/ox_inventory